09_Loading_Elf/02_Loading_And_Running.md

```diff
@@ -14,14 +14,48 @@ In the previous chapter we looked at the details of loading program headers, but
 
 - First a copy of the ELF file to be loaded is needed. The recommended way is to load a file via the VFS, but it could be a bootloader module or even embedded into the kernel.
 - Then once the ELF is loaded, we need verify that its header is correct. Also check the architecture (machine type) matches the current machine, and that the bit-ness is correct (dont try to run a 32-bit program if you dont support it!).
-- Find all the loadable program headers for the ELF, we'll need those in a moment.
+- Find all the loadable program headers for the ELF, we'll need those in a moment (They are the headers with type `PT_LOAD`).
```

Comment:
I think rather than add this info in brackets we could just reword the sentence to easier to understand :) Perhaps:
> Find all the program headers with the type `PT_LOAD`, we'll need those in a moment.